lib/gitter-adapter.js

```diff
@@ -148,6 +148,14 @@ Adapter.prototype.sendMessage = function(target, message) {
   var uri = target.replace('#','');
   var isStatus = /^\u0001ACTION/.test(message);
 
+  var regResult = /^((\w+)[:,]\s*).+/.exec(message);
+  var mention, mentioned;
+
+  if (regResult !== undefined) {
```

Comment:
Damn you're right, I had trouble running the server locally, I even checked the result of `.exec` when it doesn't find a match, woops :sweat_smile: